test/src/e2e_vm_tests/test_programs/should_fail/contract_dependencies_conflicting_salt/contract_a/src/main.sw

```diff
@@ -1,16 +1,12 @@
 contract;
 
-use contract_b::CONTRACT_ID as CONTRACT_B_ID;
-use contract_c::CONTRACT_ID as CONTRACT_C_ID;
-
 abi MyContract {
     fn test_function();
 }
 
 impl MyContract for Contract {
     fn test_function() {
-        let CONTRACT_B = CONTRACT_B_ID;
-        let contract_b_id = ContractId::from(CONTRACT_B);
+        let contract_b_id = ContractId::from(contract_b::CONTRACT_B);
```

Comment:
woops
```suggestion
        let contract_b_id = ContractId::from(contract_b::CONTRACT_ID);
```

Reply:
Looks like I should stop working at 3 AM.. 😵‍💫